.goreleaser.yml

```diff
@@ -49,3 +49,35 @@ nfpm:
   formats:
     - deb
     - rpm
+
+brew:
+  name: exo
+
+  github:
+    owner: pierre-emmanuelJ
```

Comment:
Shouldn't the owner be the exoscale org?

Reply:
It's was WIP